chart/values.yaml

```diff
@@ -44,6 +44,17 @@ jsonLogs: true
 # restrict canary-checker to monitor single namespace for canaries.  Leave blank to monitor all namespaces
 canaryNamespace: ""
 
+# Only canaries matching these filters will run.
+# If left empty, all canaries will run.
+canaryFilter:
```

Comment:
can we use canaryNamespaceSelector and canarySelector to be more consistent with prometheus